packages/jest-worker/src/index.ts

```diff
@@ -124,7 +124,7 @@ export class Worker {
         return;
       }
 
-      if (this.constructor.prototype.hasOwnProperty(name)) {
+      if (Object.prototype.hasOwnProperty.call(this.constructor, name)) {
```

Comment:
@SimenB this one seems to be the root of error.

Reply:
`Object.prototype.constructor(name)`

Reply:
let me try pushing this one

Reply:
oh i looked up the rule and found this [](https://eslint.org/docs/rules/no-prototype-builtins#when-not-to-use-itl)

Reply:
disabling seems fine 🙂